### PR TITLE
IE checkbox/exclamation icons

### LINKED ISSUE #2510: IE11 - green check icons too small
- [x] icons are too small
screen shot 2018-01-25 at 3 28 14 pm

### PR DESCRIPTION
**Pull request by floresj**
Resolves #2510

### CODE REVIEW

File: src/img/checkmark.svg [hunk 1]
```diff
@@ -1,15 +1,10 @@
...
 <!-- Generator: Adobe Illustrator 20.1.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
 <svg version="1.1" id="Alerts" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
-	 viewBox="0 0 15.95 15.96" style="enable-background:new 0 0 15.95 15.96;" xml:space="preserve">
-<style type="text/css">
-	.st0{fill:#47A362;}
-</style>
-<g>
-	<path class="st0" d="M7.98,15.96C3.57,15.96,0,12.39,0,7.98C0,3.57,3.57,0,7.98,0s7.98,3.57,7.98,7.98
+	 viewBox="0 0 15.95 15.96" xml:space="preserve">
+	<path fill="#4aa564" d="M7.98,15.96C3.57,15.96,0,12.39,0,7.98C0,3.57,3.57,0,7.98,0s7.98,3.57,7.98,7.98
```
Comment (floresj) [inline src/img/checkmark.svg]: Adding fill allowed svgs to show up with color in FF. They were coming back black when used as a bgimage.

File: src/components/Navigation/Navigation.scss [hunk 2]
```diff
@@ -60,7 +60,7 @@
       height: 100%;
       width: 1.7rem;
       background: center center no-repeat;
-      background-size: contain;
+      background-size: auto;
```
Comment (floresj) [inline src/components/Navigation/Navigation.scss]: This was slightly cutting off icon in navigation headers in IE.

File: src/views/Form/Form.scss [hunk 3]
```diff
@@ -95,14 +95,14 @@ input[type='checkbox'], input[type='radio'] {
   input[type='email'],
   textarea,
   select {
-    background: url('../img/exclamation-point.svg') no-repeat right 0.7rem center / 1.7rem auto;
+    background: url('../img/exclamation-point.svg') no-repeat right 0.7rem center / 1.7rem 1.7rem;
```
Comment (floresj) [inline src/views/Form/Form.scss]: This was input error icons to not show up.

File: src/views/Form/Form.scss [hunk 4]
```diff
@@ -95,14 +95,14 @@ input[type='checkbox'], input[type='radio'] {
   input[type='email'],
   textarea,
   select {
-    background: url('../img/exclamation-point.svg') no-repeat right 0.7rem center / 1.7rem auto;
+    background: url('../img/exclamation-point.svg') no-repeat right 0.7rem center / 1.7rem 100%;
```
Comment (floresj) [inline src/views/Form/Form.scss]: This was causing input error icons to not show up in IE.

File: src/views/Form/Form.scss [hunk 5]
```diff
@@ -95,14 +95,14 @@ input[type='checkbox'], input[type='radio'] {
...
     box-shadow: none;
     border-width: 2px;
     width: 100%;
 
     &.usa-input-success {
       box-shadow: none;
-      background: url('../img/exclamation-point.svg') no-repeat right 0.7rem center / 1.7rem auto;
+      background: url('../img/exclamation-point.svg') no-repeat right 0.7rem center / 1.7rem 100%;
```
Comment (floresj) [inline src/views/Form/Form.scss]: This was causing input error icons to not show up in IE.

File: src/img/checkmark.svg (hunk 1 above)
Comment (bmallred) [reply src/img/checkmark.svg]: Tried to reproduce on my end but Firefox crashes immediately on the site. Taking your word for it :stuck_out_tongue_closed_eyes:

Comment (floresj) [reply src/img/checkmark.svg]: Noooooo! We'll let Jay take this one :stuck_out_tongue:

**Comment by bmallred** [comment]
So note to self, IE still sucks on SVGs.

**Pull request merged**